### PR TITLE
Adds scripting and DB edits for NM Backoo

### PR DESCRIPTION
**Pull request by tankfest**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

I'll be doing live testing on Gold Saucer over the next few days for this; holding here for critiquing as well.

Thanks to cocosolos for assistance with spawn conditions!

### CODE REVIEW

File: scripts/zones/Buburimu_Peninsula/Zone.lua [hunk 1]
```diff
@@ -4,60 +4,81 @@
...
 function onChocoboDig(player, precheck)
     return tpz.chocoboDig.start(player, precheck)
-end;
+end
 
 function onInitialize(zone)
+    local hour = VanadielHour()
+    
+    if hour >= 6 and hour < 16 then
+        DisallowRespawn(ID.mob.BACKOO, false)
```
Comment (cocosolos) [inline scripts/zones/Buburimu_Peninsula/Zone.lua]: Should just need to change this to `GetMobByID(ID.mob.BACKOO):setRespawnTime(1)`

File: scripts/zones/Buburimu_Peninsula/Zone.lua [hunk 2]
```diff
@@ -4,60 +4,81 @@
...
 function onRegionEnter(player,region)
-end;
+end
+
+function onGameHour(zone)
+    local hour = VanadielHour()
+    local nmBackoo = GetMobByID(ID.mob.BACKOO)
+
+    if hour == 6 then -- backoo time-of-day pop condition open
+        DisallowRespawn(ID.mob.BACKOO, false)
```
Comment (cocosolos) [inline scripts/zones/Buburimu_Peninsula/Zone.lua:58]:
```lua
local backoo = GetMobByID(ID.mob.BACKOO)
if backoo:getRespawnTime() == 0 then
    backoo:setRespawnTime(1)
else
    DisallowRespawn(ID.mob.BACKOO, false)
end
```

**Review by cocosolos — APPROVED** [review]

File: sql/mob_groups.sql [hunk 3]
```diff
@@ -7927,6 +7927,7 @@ INSERT INTO `mob_groups` VALUES (51,3148,118,'Pixie',0,128,2001,0,0,51,54,0);
 INSERT INTO `mob_groups` VALUES (52,2213,118,'Ketos',1,0,0,0,0,1,1,0);
 INSERT INTO `mob_groups` VALUES (53,4734,118,'Botulus_Rex',0,128,0,0,9999,94,95,0);
 INSERT INTO `mob_groups` VALUES (54,1642,118,'Goblin_Bounty_Hunter',0,0,1030,0,0,17,20,0);
+INSERT INTO `mob_groups` VALUES (55,5736,118,'Backoo',0,128,3140,2500,0,37,37,0);
```
Comment (ibm2431) [inline sql/mob_groups.sql:7930]: Might need a different pool. This is the pool for Koropokkur, which has a skill list of _only_ Wild Oats.

**Comment by tankfest** [comment]
It should be noted that its spawn position input in the database is a rough guesstimate based on what ffxiclopedia had to say.  Could use a capture (should be fairly easy given his regular spawn timing) to determine more accurate spawn coords.

edit: Nyu provided a retail capture for his position, thanks!

File: sql/mob_groups.sql (hunk 3 above)
Comment (tankfest) [reply sql/mob_groups.sql:7930]: Swapped him to Tom Tit Tat's pool, who uses no special abilities outside of standard mandy moves.

Comment (TeoTwawki) [reply sql/mob_groups.sql:7930]: but then its using all the same pool data as Tom, which may not be correct. More than likely a flag value will be different (probably size bits) and those are stored in pool. Recommend just making a new pool for new mobs.

**Pull request merged**